qa-tests-backend/src/test/groovy/AdmissionControllerTest.groovy

```diff
@@ -217,11 +217,7 @@ class AdmissionControllerTest extends BaseSpecification {
         "Suppress CVE and check that the deployment can now launch"
 
         def cve = "CVE-2019-3462"
-        if (Env.CI_JOBNAME.contains("postgres")) {
-            CVEService.suppressImageCVE(cve)
-        } else {
-            CVEService.suppressCVE(cve)
-        }
+        CVEService.suppressImageCVE(cve)
```

Comment:
I believe this is not a part of centralize postgres run identification logic

Reply:
The `if (Env.CI_JOBNAME.contains("postgres")) {...} else {...}` block actually duplicate the code of `CVEService.suppressImageCVE` (same for unsuppress).
I can move the change somewhere else if this is too confusing here.